yaksh/test_views.py

```diff
@@ -1072,6 +1073,57 @@ def test_course_detail_get(self):
         self.assertTemplateUsed(response, 'yaksh/course_detail.html')
 
 
+    def test_send_mail_to_course_students(self):
+        """ Check if bulk mail is sent to multiple students enrolled in a course
+        """
+        self.client.login(
+            username=self.user1.username,
+            password=self.user1_plaintext_pass
+        )
+        self.student2 = User.objects.create_user(
+            username='demo_student2',
+            password=self.student_plaintext_pass,
+            first_name='student_first_name',
+            last_name='student_last_name',
+            email='demo_student2@test.com'
+        )
+        self.student3 = User.objects.create_user(
+            username='demo_student3',
+            password=self.student_plaintext_pass,
+            first_name='student_first_name',
+            last_name='student_last_name',
+            email='demo_student3@test.com'
+        )
+        self.student4 = User.objects.create_user(
+            username='demo_student4',
+            password=self.student_plaintext_pass,
+            first_name='student_first_name',
+            last_name='student_last_name',
+            email='demo_student4@test.com'
+        )
+        user_ids = [self.student.id, self.student2.id, self.student3.id,
```

Comment:
List comprehension will lead to unnecessary for loop which is not good according to me.